src/apps/common/ns_turn_utils.c

```diff
@@ -709,7 +709,7 @@ int get_canonic_origin(const char *o, char *co, int sz) {
               }
             }
 
-            int port = evhttp_uri_get_port(uri);
+            uint16_t port = evhttp_uri_get_port(uri);
             if (port < 1) {
```

Comment:
Now that `port` is uint16_t, this would be more meaningfully expressed as `port != 0`